Signal/src/util/OWSScreenLockUI.m

```diff
@@ -372,14 +371,14 @@ - (void)showScreenLockFailureAlertWithMessage:(NSString *)message
                          // After the alert, update the UI.
                          [self ensureUI];
                      }
-               fromViewController:self.screenBlockingViewController];
+               fromViewController:self.screenBlockingWindow.rootViewController];
```

Comment:
Just for my understanding - this is *no* change in behavior, right?

Reply:
Correct.